reference-architecture/azure-ansible/logging.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+USERNAME=$1
+PASSWORD=$2
+HOSTNAME=$3
+NODECOUNT=$4
+ROUTEREXTIP=$5
+RHNUSERNAME=$6
+RHNPASSWORD=$7
+RHNPOOLID=$8
+SSHPRIVATEDATA=$9
+SSHPUBLICDATA=$10
+SSHPUBLICDATA2=$11
+SSHPUBLICDATA3=$12
+
+
+mkdir -p /home/$USERNAME/.ssh
+echo $SSHPUBLICDATA $SSHPUBLICDATA2 $SSHPUBLICDATA3 >  /home/$USERNAME/.ssh/id_rsa.pub
+echo $SSHPRIVATEDATA | base64 --d > /home/$USERNAME/.ssh/id_rsa
+chown $USERNAME /home/$USERNAME/.ssh/id_rsa.pub
+chmod 600 /home/$USERNAME/.ssh/id_rsa.pub
+chown $USERNAME /home/$USERNAME/.ssh/id_rsa
+chmod 600 /home/$USERNAME/.ssh/id_rsa
+
+mkdir -p /root/.ssh
+echo $SSHPRIVATEDATA | base64 --d > /root/.ssh/id_rsa
+echo $SSHPUBLICDATA $SSHPUBLICDATA2 $SSHPUBLICDATA3   >  /root/.ssh/id_rsa.pub
+chown root /root/.ssh/id_rsa.pub
+chmod 600 /root/.ssh/id_rsa.pub
+chown root /root/.ssh/id_rsa
+chmod 600 /root/.ssh/id_rsa
+
+subscription-manager unregister 
+subscription-manager register --username $RHNUSERNAME --password $RHNPASSWORD
```

Comment:
Note the intention of doing it here, was to get the logging host up so it could provide logging before everything is ready. For the moment I'm deleting logging host.